benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -831,28 +836,35 @@ void GraphicsBenchmarkApp::Render()
 
     RecordCommandBuffer(frame, swapchain, imageIndex);
 
-    grfx::SubmitInfo submitInfo   = {};
-    submitInfo.commandBufferCount = 1;
-    submitInfo.ppCommandBuffers   = &frame.cmd;
+    size_t submissionCount = pSubmissionCount->GetValue();
+    mSubmissionTimes.resize(submissionCount);
+    for (size_t i = 0; i < submissionCount; i++) {
+        grfx::SubmitInfo submitInfo   = {};
+        submitInfo.commandBufferCount = 1;
+        submitInfo.ppCommandBuffers   = &frame.cmd;
```

Comment:
This is not legal in Vulkan, and triggerd VUID-vkQueueSubmit-pCommandBuffers-00071.
When you call submit, the command buffer goes from executable to pending until its execution finished (https://docs.vulkan.org/spec/latest/chapters/cmdbuffers.html#:~:text=Pending,the%20commands%20recorded%20to%20it.)
This means the next loop will call submit with a buffer having the "pending" state, which is incorrect.

Seems like adding the VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT bit is enough to solve this. I'd be tempted to add this bit by default on all primary command buffers we create with BigWheels, but unsure if this has any drawbacks. Probably perf-wise since the driver cannot assume the command-buffer remains unchanged while recording?
(@chaoticbob might know)

Reply:
Thanks for mentioning this. it is indeed a problem, and we want to avoid using VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT because there could be some extra memory allocations at driver level. I had some discussions with Shan, and will change the design of this knob a bit.